source/layouts/partials/_acquire_details.erb

```diff
@@ -16,12 +16,12 @@
                         <li style="margin-bottom: 10px;">
                             <ul style="list-style-type: none;">
                                 <li>
-                                    <a href="https://bisq.network/">
+                                    <a href="https://markets.bisq.network/?market=grc_btc">
                                         <img src="../assets/img/Exchanges/bisq.png" alt="bisq" />
                                     </a>
                                 </li>
                                 <li>
-                                    <a href="https://market.rudex.org/#/asset/RUDEX.GRC">
+                                    <a href="https://staging.rudex.org/#/market/OPEN.GRC_OPEN.BTC">
```

Comment:
This is not the correct asset pair.  Note Issue #121 by @grctest .  The OPEN.GRC gateway is controlled by OpenLedger and remains closed.  Use the RUDEX.GRC gateway instead.